sdk/translation/ai-translation-text-rest/samples-dev/dictionaryExamples.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary This sample demonstrates how to make a simple call to the Azure Text Translator service to get grammatical structure and context examples for the source term and target term pair.
+ */
+import TextTranslationClient, {
+  ErrorResponseOutput,
+  TranslatorCredential,
+  DictionaryExampleTextItem,
+  LookupDictionaryExamplesQueryParamProperties,
+  DictionaryExampleItemOutput,
+  isUnexpected,
+} from "@azure-rest/ai-translation-text";
+
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const endpoint = process.env["ENDPOINT"] || "https://api.cognitive.microsofttranslator.com";
+const apiKey = process.env["TEXT_TRANSLATOR_API_KEY"] || "<api key>";
+const region = process.env["TEXT_TRANSLATOR_REGION"] || "<region>";
+
+export async function main() {
+  console.log("== Dictionary Examples sample ==");
+
+  const translateCedential = new TranslatorCredential(apiKey, region);
+  const translationClient = TextTranslationClient(endpoint, translateCedential, undefined);
+
+  const inputText: DictionaryExampleTextItem[] = [{ text: "fly", translation: "volar" }];
+  const parameters: LookupDictionaryExamplesQueryParamProperties & Record<string, unknown> = {
+    to: "es",
+    from: "en",
+  };
+  const dictionaryResponse = await translationClient.path("/dictionary/examples").post({
+    body: inputText,
+    queryParameters: parameters,
+  });
```

Comment:
Let's inline the parameters to avoid having the intersection type with Record<string, unknown>

```suggestion
  const dictionaryResponse = await translationClient.path("/dictionary/examples").post({
    body: inputText,
    queryParameters: {
      to: "es",
      from: "en",
    }
  });
```

Reply:
This applies to other places where `& Record<string, unknown>` shows up in the samples

Reply:
ok, done